ballerina/tests/resources/expected_results/variable_with_invalid_default_value2.json

```diff
@@ -0,0 +1,13 @@
+{
+    "errors": [
+        {
+            "message": "Variable \"weight\" of type \"Float\" has invalid default value: Walter. Expected type \"Float\", found Walter",
```

Comment:
Shouldn't this be 

```
message: "Variable \"weight\" of type \"Float\" has invalid default value: Walter. Expected type \"Float\", found \"String\""
```
?

We are talking about the type here right? `Expected "Float", found Walter` doesn't seems correct.

Reply:
As per the discussion, keep it's as same.